### ISSUE TITLE
Banner colors not tinting properly

### ISSUE THREAD

**Issue by 4lch4** [TeamJeeves/Guild-Emblem-Generator#1]
# What's Wrong?

Currently, when generating an emblem the background color doesn't appear to look quite right, and isn't exactly what is specified. For example, if you use this test emblem object, it'll generate the following image:

```javascript
const testEmblem = {
  'icon': 97,
  'iconColor': 'ff101517',
  'iconColorId': 15,
  'border': 0,
  'borderColor': 'ff0f1415',
  'borderColorId': 15,
  'backgroundColor': 'fffc6891',
  'backgroundColorId': 49
}
```
![Generated_Image](https://user-images.githubusercontent.com/6931731/40271997-acb8e8c4-5b6b-11e8-88c1-476bad5587e3.png)

If you look up the background color, ignoring the first two `f` values, [it's actually](http://www.color-hex.com/color/fc6891) much more like this: 

![fc6891](https://user-images.githubusercontent.com/6931731/40272057-853d05e0-5b6c-11e8-9fb5-34c9ee352634.png)

## Why's this happening?

I'm fairly certain this is because when generating the image we mix the default banner color with the desired color when we tint that part of the image and then it doesn't quite look right in the end.

In short, the [Modifier](../blob/master/util/Modifier.js) class needs updating to properly tint the image instead of mixing the colors.
